docs/pages/management/operations/proxy-peering.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: Proxy Peering Migration
+description: How to upgrade an existing Teleport cluster to proxy peering mode.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="10.0"
+>
+  TLS routing is available starting from Teleport `10.0`.
+</Details>
+
+## Requirements
+
+Teleport Proxy servers must be able to reach each other over the network. Ensure
+there are no firewall policies that would block proxy to proxy communication.
+
+## Step 1/4. Upgrade to teleport `10.0`
+Download Teleport `10.0` or later from the [downloads page](https://goteleport.com/teleport/download) or your enterprise portal
```

Comment:
Since these instructions are environment specific, I think we should remove this section and include "Teleport 10.0" in the Prerequisites section. Otherwise, there are a lot of use cases we don't cover in this guide, e.g., Kubernetes deployments. Also, which Teleport services need to be on 10.0?

Reply:
Moved to prereqs here [593d6ac](https://github.com/gravitational/teleport/pull/15790/commits/593d6aca044d9dcfe96fd28099dbcedc9e2d431e)